packages/frontend/src/types/index.ts

```diff
@@ -93,14 +93,14 @@ export enum SortOption {
   unavailable = "Unavailable",
 }
 
-export type Store = {
-  id: number;
-  name: string;
-  thumbnail: string;
-  products: IProduct[];
-  transactions: ITransaction[];
-  contributors: IContributor[];
-  authorizedKeyCards: number[];
-  relays: IRelay[];
-  tags: ITag[];
-};
+// export type Store = {
```

Comment:
Commenting this out for TS checks, but we will eventually need this for store configuration updates